book/index.md

```diff
@@ -39,5 +38,5 @@ to be important. For instance, we do not cover sorting algorithms,
 a topic that rarely arises in practice, at all; instead, we spend more time
 walking you through interesting applications of tree and graph traversal
 algorithms. For this reason, we recommend that you work through the examples
-slowly, with a REPL or text editor open, verifying that you absorb the content
-at the depth required to appreciate its timeless utility.
```

Comment:
I always thought this was super dramatic, so forgive me for replacing it with something more down-to-earth. :earth_americas: